templates/cli/lib/commands/generic.js.twig

```diff
@@ -13,7 +13,8 @@ const DEFAULT_ENDPOINT = 'https://cloud.appwrite.io/v1';
 
 const loginCommand = async ({ email, password, endpoint, mfa, code }) => {
     const oldCurrent = globalConfig.getCurrentSession();
-    let configEndpoint = endpoint ?? DEFAULT_ENDPOINT;
+
+    const configEndpoint = endpoint ?? globalConfig.getEndpoint() ?? DEFAULT_ENDPOINT;
```

Comment:
Does it mean it will use the last endpoint?

Reply:
So it will select, in order of preference:
 - endpoint set with --endpoint on command
 - endpoint set with appwrite client --endpoint
 - "https://cloud.appwrite.io"